### DISCUSSION TITLE
Join OpenSubsonic?

### DISCUSSION THREAD

**Discussion by deluan** [BLeeEZ/amperfy#187] [Ideas]
Hey @BLeeEZ, sorry for opening a discussion for this, but I couldn't find a way to contact you directly. 

Not sure if you know, but some Subsonic server and client devs are joining forces to form an effort to evolve the Subsonic API: https://github.com/opensubsonic/open-subsonic-api/discussions  Are you interested in joining as a member?

**Comment by BLeeEZ** [comment]
Hi, this looks really cool. I will contact you if new adjustments to the API come to my mind.
